var/spack/repos/builtin/packages/py-jaraco-functools/package.py

```diff
@@ -0,0 +1,12 @@
+from spack import *
+
+
+class PyJaracoFunctools(PythonPackage):
+    """Functools like those found in stdlib"""
+
+    homepage = "https://github.com/jaraco/jaraco.functools"
+    url      = "https://files.pythonhosted.org/packages/a9/1e/44f6a5cffef147a3ffd37a748b8f4c2ded9b07ca20a15f17cd9874158f24/jaraco.functools-2.0.tar.gz"
```

Comment:
```suggestion
    url      = "https://pypi.io/packages/source/j/jaraco.functools/jaraco.functools-2.0.tar.gz"
```
This will allow Spack to interpret the URL for other versions as well.